docker-maintainer/README.md

```diff
@@ -0,0 +1,94 @@
+# DOMjudge Docker container for maintainers
```

Comment:
nit: we may want to change the word maintainers here and elsewhere in DOMjudge docs to contributors.

Reply:
Seems reasonable, I'll change it.